### PR TITLE
Dressing room 2 added and avatar modified

### PR DESCRIPTION
**Pull request by aanchal07**
I have added the following: 

1. Another dressing room from which user can get clothes, hair and accessories.
2. On clicking on any feature in dressing room 2, another activity opens up in which you can select a feature variant.
2. Modified the final avatar screen that comes after you choose avatar.
3. Added landscape xmls for: avatar.xml, completed_scene.xml, dressing_room.xml, select_feature.xml.
screenshot_2016-06-02-22-45-57
layout-2016-06-04-010015
layout-2016-06-04-005940
layout-2016-06-04-010142

### CODE REVIEW

File: PowerUp/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,31 +18,28 @@
...
+            android:label="@string/app_name" />
         <activity
             android:name=".GameOverActivity"
-            android:label="@string/app_name" >
-        </activity>
+            android:label="@string/app_name" />
 
         <uses-library android:name="android.test.runner" />
+
+        <activity android:name=".DressingRoomActivity"></activity>
```
Comment (chhavip) [inline PowerUp/app/src/main/AndroidManifest.xml]: Give each activity a label

**Comment by chhavip** [comment]
@aanchal07 Good work done, a few pointers:
1) The powers shown is a static image I believe? What is the use of the same and for consistency's sake the same look and functionality should be provided throughout
2) Have left some comments about the coding style
3) Your PR has conflicts with the base branch, please resolve them and resubmit

**Comment by aanchal07** [comment]
@chhavip Thanks for looking into it. :)

Yes the power shown is a static image as we were focusing mainly on the dressing room features in this week. I will add its implementation in the next week. 

Also I need little help with the PR conflict thing.

Thanks!

**Comment by aanchal07** [comment]
@chhavip Thank you for your help!
I have added the changes in manifest and resolved the conflict. I hope it is fine now. :)

**Comment by chhavip** [comment]
@aanchal07 Good work on getting the conflicts out. You would need to submit this one feature as one commit. For that refer to squashing commits. For future references I suggest submitting one feature as one commit, let me know if you need help with that.

**Comment by aanchal07** [comment]
@chhavip Ok thank you. :)
Do I need to make a new branch for this and then squash the commits?

**Comment by chhavip** [comment]
@aanchal07 It can be done in the same branch, creating a new branch would require you to copy the entire work again. https://github.com/chhavip/Git-Guide#squashing-commits. See if this helps you.

**Comment by aanchal07** [comment]
@chhavip When I squashed the 4 commits into one, there was a conflict with the base branch so I had to resolve that conflict and make another commit. Now when I try to squash these two commits, I get the conflict again. I have tried over 3-4 times but no success. :(

**Comment by chhavip** [comment]
Okay, so we can do this two ways:
1) You create a new clean branch and copy your previous work to it in one commit
2) We can try Remote Login where I try to solve the commit issue on your machine.
Advice: Keep all the files you have made changes to in a local copy separate from your clone in case it gets written over.

**Comment by aanchal07** [comment]
Thank you for all the help!
1) Creating a new branch and then adding the previous committed work by cherry pick?
2) We can use Team viewer for the remote login if possible.

**Comment by chhavip** [comment]
I was actually suggesting to just copy the files with the changes locally in a separate folder, then create a new branch and then copy back the changes in this branch. You can go the cherrypick route if you are comfortable with that.

**Pull request closed without merging**